README.md

```diff
@@ -38,9 +38,38 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+## Trello Setup
+
+### Environment Variables
+
+In `.env`, add your Trello api key and token. If you don't have a key or token, follow the instructions [here](https://developer.atlassian.com/cloud/trello/guides/rest-api/api-introduction/#managing-your-api-key) to set them up:
+
+```bash
+$ TRELLO_API_KEY=<key>
+$ TRELLO_API_TOKEN=<token>
+```
```

Comment:
Thanks for these clear instructions! A nice addition to this it to add the text above to the `.env.template` file, so that there's another prompt there